src/documents/management/commands/document_importer.py

```diff
@@ -77,7 +77,7 @@ def _check_manifest(self):
                 )
 
             doc_file = record["__exported_file_name__"]
-            if not os.path.exists(os.path.join(self.source, doc_file)):
+            if not os.path.exists(doc_file):
```

Comment:
That's interesting.  Your change makes sense given the code in the exporter, but I'm not sure that the `__exported_file_name__` value is correct in the first place.  One should be able to export from one system with one directory structure to an entirely different system without things exploding, so if we're storing the full path in the export, that's a problem to me.

I can't test this right now as I'm out of town visiting family, but if you're tinkering in this area anyway, I'd like to know if you can do an export to one directory and an import from another one or if we have a problem in the manifest file here.